recipes/tramway/meta.yaml

```diff
@@ -0,0 +1,61 @@
+
+{% set name = "tramway" %}
+{% set version = "0.5" %}
+
+package:
+  name: {{ name }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/DecBayComp/TRamWAy/archive/v{{ version }}.tar.gz
+  sha256: 6e26acc4280dfb6d93f1d5009009927dd1fa2e63f18a2c4e160b9a1e40534bc6
+
+build:
+  skip: True  # [py<36]
+  number: 0
+  # remove polytope as soon as the package is available in some conda channel, if it ever is
+  script: "{{ PYTHON }} -m pip install --no-deps https://pypi.io/packages/source/p/polytope/polytope-0.2.3.tar.gz . -vv"
```

Comment:
This should be installing from the current directory

Reply:
I guess you mean "from the current directory ONLY" ><
As mentioned in the PR message, this is indeed an issue we have with this polytope dependency that cannot be met within conda.
May I apply for maintaining this polytope package in conda? I can get in touch with the authors and seek for their agreement...

Reply:
You don’t need to be a developer of the package to maintain the recipe on conda-forge. The only real requirement is that the package license permits redistribution (any popular OSS license will).